packages/dm-core-plugins/src/list/ListPlugin.tsx

```diff
@@ -107,6 +107,9 @@ export const ListPlugin = (props: IUIPlugin & { config?: TListConfig }) => {
 
   const [showModal, setShowModal] = useState<boolean>(false)
   const [expanded, setExpanded] = useState<Record<string, boolean>>({})
+  const [hasBeenExpanded, setHasBeenExpanded] = useState<
+    Record<string, boolean>
+  >({})
```

Comment:
Think you can accomplish all this "hasBeenExpanded" stuff mush easier with [lazy](https://react.dev/reference/react/lazy)

Reply:
Hmm nay, it seems lazy is just for whole modules, not individual items? 
I might be mistaken, but I'm not sure this will help us here.